main.go

```diff
@@ -135,6 +135,7 @@ func mainError() error {
 	daemonCommand := newCommand.DaemonCommand().CobraCommand()
 
 	daemonCommand.PersistentFlags().String(f.Service.Deployer.Environment, "", "Environment name that draughtsman is running in.")
+	daemonCommand.PersistentFlags().String(f.Service.Deployer.Provider, "", "Provider that draughtsman is running in.")
```

Comment:
How is that injected here? I think you'd need to modify the helm chart as well.

Reply:
I think (after some research) we are using a configmap and secret from installations. They are mounted as config files and then `viper` reads them and parses them into flags.

That's how `projectList` got here before.

Reply:
See: https://github.com/giantswarm/installations/pull/1250

Reply:
I see. I left a comment in the other PR.

Reply:
Resolved in the other PR.